py/occampy3/wrappers/sbm_manager.py

```diff
@@ -16,3 +16,18 @@ def __init__(self, ref=None):
 
     def make_model(self, model_name: str, make_project: bool) -> Model:
         return Model(self._ref.makeSbModel(model_name, make_project))
+
+    def getTestSampleSz(self) -> double:
+        return self._ref.getTestSampleSize()
+
+    def isValuesAreFunctions(self) -> bool:

```

Comment:
I think it would be best to have these be properties, as I think it's a bit cleaner to say `if manager.valuesAreFunctions` as opposed to `if manager.isValuesAreFunctions()`.